src/elements/predictor/script.js

```diff
@@ -107,6 +111,76 @@ async function afterAppend() {
             );
             updateView();
         });
+        $("#predictor-nextac-button").click(async function() {
+            $("#predictor-nextac-warning").alert("close");
+
+            const myData = contest.standings.StandingsData.filter(x => x.UserScreenName === userScreenName)[0];
+            if (!myData) return;
+
+            const myTaskResults = myData.TaskResults;
+            const myACTaskScreenNames = Object.keys(myTaskResults).filter(taskScreenName => myTaskResults[taskScreenName].Status === 1);
+            const myACTaskAssignments = myACTaskScreenNames.map(taskScreenName => {
+                return model.tasks.filter(task => task.taskScreenName === taskScreenName)[0].assignment;
```

Comment:
[`Array.prototype.find`](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Array/find) を使うと良さそうに見えます
```suggestion
                return model.tasks.find(task => task.taskScreenName === taskScreenName).assignment;
```